docs/user-guide/untilIdentityChanged.md

```diff
@@ -0,0 +1,61 @@
+# untilIdentityChanged
```

Comment:
@gabrielittner what do you think about this example and explanation about `untilIdentityChanged`?

Reply:
Looks good to me 👍